ui/app/AppLayouts/Profile/Sections/NotificationsContainer.qml

```diff
@@ -116,7 +116,7 @@ ScrollView {
 
             StatusSettingsLineButton {
                 //% "Play a sound when receiving a notification"
-                text: qsTrId("play-a-sound-when-receiving-a-notification")
+                text: qsTr("play-a-sound-when-receiving-a-notification")
```

Comment:
This one doesn't need to be changed

Reply:
If it appears as the ID, that's because `lrelease` wasn't run previously along with `lupdate`. We can always run it now since it causes some issues.